### ISSUE TITLE
[10.0.0] Some E2E tests are failing on mobile

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#8749]
### Description

Each of the non-passed tests was run one at a time in the incognito mode.

### List of failing tests with a breakdown by the environment in which they occur

**Browsers:** Safari and Chrome 93.0.4557.78, **OS:** 14.7.1, **Device:** iPhone SE 2020
**Browser:** Chrome 92.0.4515.166, **OS:** Android 11; SM-A51 Build/RPA1A.200720.012, **Device:** Samsung Galaxy A51
```
Selection > should show both selection handles after drag & drop
Failed: undefined is not an object (evaluating 'target.getBoundingClientRect')
error properties: Object({ __$rinfo: Object({ ctorName: 'Error', sourceURL: Object({ get: Function }), stack: Object({ get: Function }) }) })
triggerTouchEvent@http://10.0.0.2:8005/test/dist/helpers.entry.js:7851:28
_callee$@http://10.0.0.2:8005/test/dist/mobile.entry.js:4983:30
tryCatch@http://10.0.0.2:8005/test/dist/mobile.entry.js:3061:44
invoke@http://10.0.0.2:8005/test/dist/mobile.entry.js:3290:30
asyncGeneratorStep@http://10.0.0.2:8005/test/dist/mobile.entry.js:1669:24
_next@http://10.0.0.2:8005/test/dist/mobile.entry.js:1691:27
promiseReactionJob@[native code]
```
**Browser:** Chrome 93.0.4557.78 **OS:** 14.7.1, **Device:** iPhone SE 2020
```
Events > should "preventDefault" only the second "touchend" event while double-tapping (issue #7824)
Expected true to be false.
error properties: Object({ __$rinfo: Object({ ctorName: 'Error', sourceURL: Object({ get: Function }), stack: Object({ get: Function }) }) })
<Jasmine>
http://10.0.0.2:8005/test/dist/mobile.entry.js:4659:48
http://10.0.0.2:8005/test/dist/mobile.entry.js:4196:41
<Jasmine>
```

**Browser:** Safari Version/14.1.2 Mobile/15E148 Safari/604.1 **iPadOS:** 14.8, **Device:** iPad 6th generation, model num: MR7F2D/A
```
Selection > should show selection handles
TypeError: undefined is not an object (evaluating 'topLeftSelectionHandle.style') in http://10.0.0.2:8005/test/dist/mobile.entry.js (line 4965)
error properties: Object({ __$rinfo: Object({ ctorName: 'Error', sourceURL: Object({ get: Function }), stack: Object({ get: Function }) }) })
http://10.0.0.2:8005/test/dist/mobile.entry.js:4965:34
http://10.0.0.2:8005/test/dist/mobile.entry.js:4196:41
<Jasmine>
```
```
Selection > should show both selection handles after drag & drop
Failed: undefined is not an object (evaluating 'target.getBoundingClientRect')
error properties: Object({ __$rinfo: Object({ ctorName: 'Error', sourceURL: Object({ get: Function }), stack: Object({ get: Function }) }) })
triggerTouchEvent@http://10.0.0.2:8005/test/dist/helpers.entry.js:7851:28
_callee$@http://10.0.0.2:8005/test/dist/mobile.entry.js:4983:30
tryCatch@http://10.0.0.2:8005/test/dist/mobile.entry.js:3061:44
invoke@http://10.0.0.2:8005/test/dist/mobile.entry.js:3290:30
asyncGeneratorStep@http://10.0.0.2:8005/test/dist/mobile.entry.js:1669:24
_next@http://10.0.0.2:8005/test/dist/mobile.entry.js:1691:27
promiseReactionJob@[native code]
```
```
Selection > should not call the `select` method on the "focusable" textarea when selecting a cell
Expected spy select not to have been called.
error properties: Object({ __$rinfo: Object({ ctorName: 'Error', sourceURL: Object({ get: Function }), stack: Object({ get: Function }) }) })
<Jasmine>
_callee2$@http://10.0.0.2:8005/test/dist/mobile.entry.js:5020:65
tryCatch@http://10.0.0.2:8005/test/dist/mobile.entry.js:3061:44
invoke@http://10.0.0.2:8005/test/dist/mobile.entry.js:3290:30
asyncGeneratorStep@http://10.0.0.2:8005/test/dist/mobile.entry.js:1669:24
_next@http://10.0.0.2:8005/test/dist/mobile.entry.js:1691:27
http://10.0.0.2:8005/test/dist/mobile.entry.js:1698:12
Promise@[native code]
http://10.0.0.2:8005/test/dist/mobile.entry.js:1687:23
http://10.0.0.2:8005/test/dist/mobile.entry.js:4196:41
<Jasmine>
```
Below test fails also on **Browser:** Chrome 92.0.4515.166, **OS:** Android 11; SM-A51 Build/RPA1A.200720.012, **Device:** Samsung Galaxy A51
```
Text Editor > should save text value after click "Done" on iOS (focusout event)
Expected '' to be 'test'.
error properties: Object({ __$rinfo: Object({ ctorName: 'Error', sourceURL: Object({ get: Function }), stack: Object({ get: Function }) }) })
<Jasmine>
_callee$@http://10.0.0.2:8005/test/dist/mobile.entry.js:5073:42
tryCatch@http://10.0.0.2:8005/test/dist/mobile.entry.js:3061:44
invoke@http://10.0.0.2:8005/test/dist/mobile.entry.js:3290:30
asyncGeneratorStep@http://10.0.0.2:8005/test/dist/mobile.entry.js:1669:24
_next@http://10.0.0.2:8005/test/dist/mobile.entry.js:1691:27
http://10.0.0.2:8005/test/dist/mobile.entry.js:1698:12
Promise@[native code]
http://10.0.0.2:8005/test/dist/mobile.entry.js:1687:23
http://10.0.0.2:8005/test/dist/mobile.entry.js:4196:41
<Jasmine>
```

**Browser:** Chrome 87.0.4280.163, **iPadOS:** 14.8, **Device:** iPad 6th generation, model num: MR7F2D/A
```
Events > should "preventDefault" only the second "touchend" event while double-tapping (issue #7824)
Expected true to be false.
error properties: Object({ __$rinfo: Object({ ctorName: 'Error', sourceURL: Object({ get: Function }), stack: Object({ get: Function }) }) })
<Jasmine>
http://10.0.0.2:8005/test/dist/mobile.entry.js:4659:48
http://10.0.0.2:8005/test/dist/mobile.entry.js:4196:41
<Jasmine>
```

### Your environment
* Handsontable version: 10.0.0
* Browser Name and version: Safari & Chrome
* Operating System: iOS 14.7.1, ipadOS 14.8

**Comment by aninde** [comment]
The above tests still do not pass on v11.

**Comment by aninde** [comment]
Fixed for v11.1.0. All mobile tests go green.
Tested on 
- iPhone 6S (iOS 14.8) Safari, Chrome, Firefox
- Xiaomi Mi A2 Lite (Android 9), Safari, Chrome, Firefox, Edge,
- Samsung Galaxy A51 (Android 11), Safari, Chrome, Firefox, Edge, Opera, Samsung Internet, QQ browser

**Comment by AMBudnik** [comment]
Issue closed as fixed in Handsontable 11.1.0
